src/Main.js

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import Error from "./Error";
+import Resturants from "./Resturants";
+import { Button, Container, Form } from "react-bootstrap";
+import axios from "axios";
+
+class Main extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      displayInfo: false,
+      city: '',
+      cityData: {},
+      restresturantData: [],
+      locationData: [],
+      weatherData: [],
+      errorData: false
+    }
+  }
+
+  handleSearchInput = e => {
+    let cityName = e.target.value;
+    this.setState({
+      city: cityName
+    },
+      () => console.log(this.state.city)
```

Comment:
You could probably remove any console logs that aren't 100% necessary in the production build